fastlane/lanes/build.rb

```diff
@@ -24,9 +26,7 @@
 
       UI.important("Building version #{current_release_version} (#{current_build_code}) for upload to Release Channel")
 
-      unless options[:skip_confirm]
-        UI.user_error!('Aborted by user request') unless UI.confirm('Do you want to continue?')
-      end
+      UI.user_error!('Aborted by user request') if !options[:skip_confirm] && !UI.confirm('Do you want to continue?')
```

Comment:
What do you think of using positive logic here and in the other instances?

```suggestion
      UI.user_error!('Aborted by user request') unless options[:skip_confirm] || UI.confirm('Do you want to continue?')
```

Reply:
Ah, nice one. RuboCop changed it to avoid nesting, I believe, but the `unless` with a positive condition is more readable.